src/scss/override/_forms.scss

```diff
@@ -173,96 +173,139 @@ select[multiple].form-control {
 //
 // Checkbox / Radio
 //
-.checkbox label,
-.radio label {
-  padding-left: 30px;
-}
 
-.checkbox input[type=checkbox],
-.radio input[type=radio] {
-  width: 20px;
-  height: 20px;
-  margin-left: -30px;
-
-  &::-ms-check {
-    border-style: solid;
-    display: inline-block;
-    border-width: 2px;
-    background-clip: border-box;
+.form-group {
+  .checkbox:first-of-type,
+  .radio:first-of-type {
+    margin-top: 20px
   }
-}
 
-.checkbox input[type=checkbox] {
-  &::-ms-check {
-    color: $win-color-type-primary;
-    border-color: $win-color-fl-60;
-    background-color: transparent;
+  .form-group-label {
+    font-size: 15px;
```

Comment:
I checked with design on this point. The intent is for this to remain 15px across all viewports, hence not using the mixin. 

An option we could create is a non-responsive argument to pass to the mixin that keeps the size constant?